recipes/xgboost/meta.yaml

```diff
@@ -6,7 +6,7 @@
 {% set xgboost_repo = environ.get('XGBOOST_REPO', 'dmlc/xgboost') %}
 {% set xgboost_branch = environ.get('XGBOOST_BRANCH', 'main') %}
 {% set build_number = environ.get('XGBOOST_BUILD_NUMBER', 1) %}
-{% set rapids_version = environ.get('RAPIDS_VER', '0.16') %}
+{% set rapids_version = environ.get('RAPIDS_VER', '22.06') %}
```

Comment:
shouldn't this be 22.10?

Reply:
Done in 0f9ef2f and c00033a

Reply:
FWIW, the `RAPIDS_VER` environment variable is set in CI. So the value we changed here is just a fallback value.

Reply:
Thanks AJ! 🙏